config.default.yaml

```diff
@@ -358,10 +358,9 @@ renewable:
 # Costs Configuration
 costs:
   year: 2030
-  version: v0.10.0
+  version: v0.11.0
```

Comment:
My feeling is that `v0.10.0 -> v.0.11.0` update is worth an additional testing. I recognise that may be just a single line of code but the improvements in `technology-data` are quite significant, and it would be great to test all the implications also going beyond the "standard" testing geography

Reply:
@danielelerede-oet thanks a lot for the explanation and the discussion today. Sounds perfectly reasonable. Just for completeness, adding also the point you mentioned that the version update has been also needed to use the fix implemented in [this PR](https://github.com/PyPSA/technology-data/pull/168) to `technology-data` included into `v.0.11.0`.